### PR TITLE
feat: add copilot gpt templates

### PR DESCRIPTION
**Pull request by huimiu**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/27751368

### CODE REVIEW

**Comment by yuqizhou77** [comment]
should we add gpt.json for the gpt basic template as well?

File: templates/common/copilot-gpt-basic/README.md [hunk 1]
```diff
@@ -1 +1,40 @@
-For testing, will be updated soon.
\ No newline at end of file
+# Overview of the Copilot GPT template
+
+## Build a Copilot GPT from OpenAPI description document
```
Comment (yuqizhou77) [inline templates/common/copilot-gpt-basic/README.md:3]: I am not sure whether we should name it as this. This template is used for two cases:
1. Basic GPT. User won't provide API description doc
2. GPT + plugin. User will provide API description for generating plugin.

File: templates/common/copilot-gpt-basic/README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+> - [Node.js](https://nodejs.org/), supported versions: 16, 18
+> - A [Microsoft 365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts).
+> - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [Teams Toolkit CLI](https://aka.ms/teamsfx-toolkit-cli)
+> - [Copilot for Microsoft 365 license](https://learn.microsoft.com/microsoft-365-copilot/extensibility/prerequisites#prerequisites)
+
+1. First, select the Teams Toolkit icon on the left in the VS Code toolbar.
+2. In the Account section, sign in with your [Microsoft 365 account](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts) if you haven't already.
+3. Create Teams app by clicking `Provision` in "Lifecycle" section.
+4. Select `Preview in Copilot (Edge)` or `Preview in Copilot (Chrome)` from the launch configuration dropdown.
+5. Open the `Copilot` app and send a prompt to trigger your GPT.
```
Comment (yuqizhou77) [inline templates/common/copilot-gpt-basic/README.md:22]: I think this step need to be updated. Looking at the recordings, user need to click "..." at the top right, and select one of their GPTs.

File: templates/common/copilot-gpt-basic/appPackage/manifest.json.tpl [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        "short": "Short description for {{appName}}",
+        "full": "Full description for {{appName}}"
+    },
+    "accentColor": "#FFFFFF",
+    "composeExtensions": [],
+    "permissions": [
+        "identity",
+        "messageTeamMembers"
+    ],
+    "validDomains": []
```
Comment (yuqizhou77) [inline templates/common/copilot-gpt-basic/appPackage/manifest.json.tpl:31]: adding "copilotGpts"?

File: templates/common/copilot-gpt-basic/README.md (hunk 1 above)
Comment (huimiu) [reply templates/common/copilot-gpt-basic/README.md:3]: Yes, this readme is just a draft version and should be updated multiple times later. I think we should focus on the e2e testing for now. I'll have the PM review the readme, and also add some feature flags later.

File: templates/common/copilot-gpt-basic/README.md (hunk 2 above)
Comment (huimiu) [reply templates/common/copilot-gpt-basic/README.md:22]: Sure, let me update this in the following PR

File: templates/common/copilot-gpt-basic/appPackage/manifest.json.tpl (hunk 3 above)
Comment (huimiu) [reply templates/common/copilot-gpt-basic/appPackage/manifest.json.tpl:32]: Added!

**Review by hund030 — APPROVED** [review]

File: templates/common/copilot-gpt-basic/appPackage/manifest.json.tpl (hunk 3 above)
Comment (yuqizhou77) [reply templates/common/copilot-gpt-basic/appPackage/manifest.json.tpl:32]: should we add gpt.json and set value for copilotGpt?

Comment (huimiu) [reply templates/common/copilot-gpt-basic/appPackage/manifest.json.tpl:32]: Sure, let me update in next PR!

**Pull request merged**